source/fundamentals/crud/write-operations/embedded-arrays.txt

```diff
@@ -0,0 +1,188 @@
+===========================
+Update Arrays in a Document
+===========================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you will learn how to update arrays in a document.
+
+To update an array, you must do the following:
+
+- Specify the update you want to perform
+- Specify what array elements to apply your update to
+- Perform an update operation using these specifications
+
+Sample Document
+~~~~~~~~~~~~~~~
+
+The following sections feature examples that update this sample
+document: 
+
+.. code-block:: json
+
+    { "_id": 1, "color": "green", "qty": [8, 12, 18] }
```

Comment:
```suggestion
    { "_id": 1, "qty": [8, 12, 18] }
```

Nit: As the color field is not referenced in examples, I feel it should be removed from the page. I feel "_id" and "qty" are sufficient for demonstrating the concepts necessary to perform array updates.